OSRMTextInstructions/TokenType.swift

```diff
@@ -60,6 +63,8 @@ public enum TokenType: Int, CustomStringConvertible {
             return "direction"
         case .wayPoint:
             return "nth"
+        case .code:
+            return "code"
```

Comment:
If we ever introduce the concept of a code into the JavaScript version of this library, the JavaScript code (and all the affected translated strings) would likely say `{ref}`, not `{code}`.